### PR TITLE
20 GitHub login failing

### PR DESCRIPTION
**Pull request by sergiomarrocoli**
Possible fix for the cookie overflow error. 

Error occurs when hitting the github API to check if the user is part of the WCMC team, and the request we make is incorrect now due to changes on github's end (https://developer.github.com/changes/2020-02-10-deprecating-auth-through-query-param/).

Hopefully this fixes the error, but either way it needs updating.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -41,6 +41,9 @@ gem 'webpacker', '~> 4.x'
 gem 'google-api-client', '0.32.1'
 gem 'googleauth'
 
+gem 'ed25519', '~> 1.2'
+gem 'bcrypt_pbkdf', '~> 1'
```
Comment (lucacug) [inline Gemfile:45]: Can I check why these 2 gems are needed here?

**Review by lucacug — COMMENTED** [review]
Just left a small comment, otherwise happy for this to be merged

File: Gemfile (hunk 1 above)
Comment (sergiomarrocoli) [reply Gemfile:45]: the aren't strictly needed, they just allow ed25519 keys to be used as well as RSA keys for deploying with Capistrano.

 i don't know much about it, but ed25519 are somehow considered "better", and e.g. github's first suggestion in setting up keys is to use ed25519.

So i can just remove these gems and set up RSA keys instead, but other people might have to go through switching them around in future i guess? happy to leave or remove

Comment (lucacug) [reply Gemfile:45]: makes sense, let's keep those for future proof

**Comment by lucacug** [comment]
can we change the base branch to develop and then create a release rather than merging straight into master, please? you can follow this [link](https://github.com/unepwcmc/wiki/wiki/How-to-create-a-release)

**Comment by sergiomarrocoli** [comment]
> can we change the base branch to develop and then create a release rather than merging straight into master, please? you can follow this [link](https://github.com/unepwcmc/wiki/wiki/How-to-create-a-release)

thanks, now pointing at develop

**Pull request merged**